app-script-payslip-sender/email_sender.gs

```diff
@@ -0,0 +1,38 @@
+function sendEmail() {
+  let excel = SpreadsheetApp.getActiveSpreadsheet();
+
+  let sheet = excel.getSheetByName("first_sheet");
+
+  let lastRow = sheet.getLastRow();
+
+  let secondLastColumn = sheet.getLastColumn() - 1;
+
+  let range = sheet.getRange(2, 2, lastRow - 1, secondLastColumn);
+
+  let values = range.getValues();
+
+  for (let i = 0; i < values.length; i++) {
+    let status;
+    let [Name, Email, Salary] = values[i];
```

Comment:
better to follow the standard naming conventions for functions/variables


https://www.30secondsofcode.org/js/s/naming-conventions/